app/styles/app/layouts/billing.scss

```diff
@@ -155,6 +155,108 @@
     margin-bottom: 1em;
     margin-top: 1em;
 
+    &__addons_box {
+      background-color: #fff;
+      border-radius: 5px;
+      padding: 6px 13px;
+      border: 1px solid $pebble-grey;
+      margin-bottom: 10px;
+      width: 208px;
+      height: 114px;
+      flex-basis: auto;
+      margin-right: 5px;
+
+      &--desc {
+        color: $asphalt-grey;
+        font-weight: 600;
+        font-size: 0.8em;
+        margin: 5px 0 12px 0;
+        color: #9d9d9d;
```

Comment:
This color property overwrites the one above. Is it on purpose? Also it'd be better to reference colors via variables ([cement-grey](https://github.com/travis-ci/travis-web/blob/master/app/styles/app/vars.scss#L22) in this case), because they are more meaningful and this allows to better understand what color stands behind it.

Reply:
Oops, right. I've just copied values from zeplin as is, will change to variables.